src/components/capture/create/Spec.tsx

```diff
@@ -36,7 +36,7 @@ function NewCaptureSpec({ connectorImage }: Props) {
         },
         [connectorImage]
     );
-    const { data: connector, error } = useSelectSingle(tagsQuery, {});
+    const { data: connector, error } = useSelectSingle(tagsQuery);
```

Comment:
This change is in a few places. Removed requiring this field as it will probably end up normally being empty.